server/schedule/operator.go

```diff
@@ -45,6 +46,25 @@ func (tl TransferLeader) IsFinish(region *core.RegionInfo) bool {
 	return region.Leader.GetStoreId() == tl.ToStore
 }
 
+// Influence calculates the store difference that current step make
+func (tl TransferLeader) Influence(OpInfluence map[uint64]*StoreDiff, region *core.RegionInfo) {
+	from, ok := OpInfluence[tl.FromStore]
+	if !ok {
+		OpInfluence[tl.FromStore] = &StoreDiff{}
+		from = OpInfluence[tl.FromStore]
+	}
+	to, ok := OpInfluence[tl.ToStore]
+	if !ok {
+		OpInfluence[tl.ToStore] = &StoreDiff{}
+		to = OpInfluence[tl.ToStore]
+	}
```

Comment:
I think we can define a type for `map[uint64]*StoreDiff` then it can have some methods, like `GetStoreDiff(id) *StoreDiff`.